docs/content/workflows/_index.en.md

```diff
@@ -16,7 +16,7 @@ Fission Workflows requires the following components to be installed on your loca
 Fission Workflows is deployed on top of a Kubernetes cluster.
 If you don't have a Kubernetes cluster, [here's a quick guide to set one up](../kubernetessetup).
 It also requires a [Fission](https://github.com/fission/fission) deployment to be present on your Kubernetes cluster. 
-If you do not have a Fission deployment, follow [Fission's installation guide](../install).
+If you do not have a Fission deployment, follow [Fission's installation guide](../installation).
```

Comment:
`../installation/installation/` is better, cause it links to fission installation page directly